.taprc

```diff
@@ -1,9 +1,3 @@
-ts: false
-jsx: false
-flow: false
-jobs: 1
-coverage: true
-check-coverage: true
-
+disable-coverage: true
```

Comment:
I can't test Redis right now locally, so can you try removing this line to see if the coverage will pass?

Reply:
Before doing that, we also need to change istanbul-ignore with `/* c8 ignore next */` here:

https://github.com/fastify/fastify-redis/blob/904b2dfa25b0c3e30e192f8c15179ac98de52f7c/index.js#L70
https://github.com/fastify/fastify-redis/blob/904b2dfa25b0c3e30e192f8c15179ac98de52f7c/index.js#L91

Reply:
Done

Reply:
```js
/* c8 ignore start */
  const onEnd = function (err) {
    client
      .off('ready', onReady)
      .off('error', onError)
      .off('end', onEnd)
      .quit()

    next(err)
  }
/* c8 ignore stop */
```

Reply:
We need to keep coverage on as much as possible